src/containers/button-title-icon-card.html

```diff
@@ -3,10 +3,9 @@
   <h4 class="primary-content mdc-typography--headline6 text-blue text-centered">{{ title }}</h4>
   <p class="mdc-typography--body1 secondary-content text-centered">{{ text }}</p>
   <div class="mdc-card__actions centered">
-    <a class="show-me-button mdc-button mdc-button--raised mdc-card__action mdc-card__action--button"
-       href="">
+    <button class="show-me-button mdc-button mdc-button--raised mdc-card__action mdc-card__action--button nav-button">
```

Comment:
Should we rename this class from 'nav-button' to 'scroll-button'? This class could be use throughout the site, not just in the navigation buttons. Would require renaming the NavButtonUtil as well. 

Although you could take it as "this button is navigating me somewhere" and it still makes sense.

Reply:
Perhaps, but maybe that should go in a different PR. We might want to make it a little more generic too, it assumes there are all HTMLButtonElement, but I am going to use for the main <h1> in the top hero.

I also had a question about the button ripple initialization and if that should go here?

But again, these things for another issue.